### PR TITLE
add filter to plugin card description

### PR DESCRIPTION
**Pull request by afragen**
Add a filter to the plugin install list table to allow for modification of the plugin description in the plugin card.

Trac ticket: https://core.trac.wordpress.org/ticket/55480

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/includes/class-wp-plugin-install-list-table.php [hunk 1]
```diff
@@ -514,6 +514,15 @@ public function display_rows() {
 
 			// Remove any HTML from the description.
 			$description = strip_tags( $plugin['short_description'] );
+
+			/**
+			 * Filter the plugin card description.
+			 *
+			 * @since 6.0.0
+			 * @param string $decription Plugin card description.
```
Comment (dream-encode) [inline src/wp-admin/includes/class-wp-plugin-install-list-table.php]:
```suggestion
			 * @param string $description Plugin card description.
```

File: src/wp-admin/includes/class-wp-plugin-install-list-table.php [hunk 2]
```diff
@@ -514,6 +514,15 @@ public function display_rows() {
...
+			 * @param array  $plugin     Array of plugin data.
```
Comment (dream-encode) [inline src/wp-admin/includes/class-wp-plugin-install-list-table.php]:
```suggestion
			 * @param array  $plugin      Array of plugin data.
```

**Review by dream-encode — COMMENTED** [review]
Minor typo.

**Comment by afragen** [comment]
🤦🏻‍♂️ thanks for the assist 🙏🏼

File: src/wp-admin/includes/class-wp-plugin-install-list-table.php [hunk 3]
```diff
@@ -514,6 +514,15 @@ public function display_rows() {
 
 			// Remove any HTML from the description.
 			$description = strip_tags( $plugin['short_description'] );
+
+			/**
+			 * Filter the plugin card description.
+			 *
+			 * @since 6.0.0
```
Comment (peterwilsoncc) [inline src/wp-admin/includes/class-wp-plugin-install-list-table.php:521]:
```suggestion
			 * @since 6.0.0
			 *
```

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! Merged in https://core.trac.wordpress.org/changeset/53074.

**Pull request closed without merging**